ci-operator/jobs/openshift/pagerduty-short-circuiter/openshift-pagerduty-short-circuiter-main-presubmits.yaml

```diff
@@ -1,5 +1,159 @@
 presubmits:
   openshift/pagerduty-short-circuiter:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build03
```

Comment:
Why is build03 here instead of build01?

Reply:
The same was already mentioned in the previous YAML file. No changes were done to the cluster. Would confirm if it can be changed to build01